### ISSUE TITLE
LS doesn't output color in cmd (on windows 7?)

### ISSUE THREAD

**Issue by sigmap** [git-for-windows/git#1518]
I've tried this on on 64 bit and 32 bit and the issue seems to be across all of them

I can run `ls --color` and the output lacks any colorization of results. 
even running from withing usr\bin outputs to all white. 

### Setup

 - v2.16.1.windows.4 64-bit (where I first noticed the problem)
-  v2.16.2.windows 32-bit and 64bit (for test)
 - OS: Windows 7 64-bit

```
$ cmd.exe /c ver 6.1.7601

Microsoft Windows [Version 6.1.7601]

```

 - Portable install, and the package that came with cmder (v2.16.1.windows.4 64-bit)

### Details

 - Works find in gitbash, but cmd/git-cmd output lacks color

**Comment by dscho** [comment]
> Works find in gitbash, but cmd/git-cmd output lacks color

How do you call `ls.exe` in CMD? Maybe you chose non-standard install options (would have been good to fill out that issue reporting template so that that question would have been answered automatically)?

Now, while `ls.exe` is not in the `PATH` by default, there *is* a dirty trick to use `.exe` files that are included in Git for Windows only as support files (and on which users should not start to rely, because these support files might go away as soon as Git does not require them anymore): using aliases. In this case:

```cmd
git -c alias.ls=!ls ls --color
```

The output looks like this on my machine (Windows 10, though, definitely without cmder, sorry for the screenshot, copy/pasted text does not show color though):

![image](https://user-images.githubusercontent.com/127790/36733295-7972dd56-1bd0-11e8-90b4-f2dd9a97abe1.png)

**Comment by sigmap** [comment]
I tried it with `ls --color` and a few other ways. Nothing happened. I wouldn't get anything but that drab white text. However after trying a few things I decided to try toying around with the `$TERM`/`%TERM%` envar. Tried a few different things and still got white text. However unsetting `TERM` with `SET TERM=` cleared the variable and allowed me to get color.
Turns out strawberry perl sets a `TERM` envar but unsetting it by hand or somewhat automatedly in cmder/conemu will fix it. Thanks @dscho for taking the time to help.
